### PR TITLE
Add blog post "What types of applications can you build with Spice.ai?"

### PR DESCRIPTION
**Pull request by phillipleblanc**
Fixes the spacing in the v0.5 release notes post.

**What is the goal of the post and why does it need to be written?**
What types of applications can Spice.ai help make?

The goal is to educate developers on the types of applications that Spice.ai can help them to create.

**What is the intent of the post?**
Inform

**Target Audience**
Professional or hobbyist developer that is interested in using Spice.ai, but not sure how it can help them build their applications.

**Draft Outline**
- Spice.ai's architecture enables a new class of apps that can be built that learn and adapt.
- Spice.ai enables apps to make intelligent decisions. Talk about example use cases, draw a parallel with "expert systems" that have been around for decades. Make that easy.
- Spice.ai works best on real-time data streams. For training a good model, it also needs access to lots of historical data - we recognize this is a challenge for most projects and is something we are working on.

### CODE REVIEW

**Review by Corentin-pro — APPROVED** [review]
Nicely put, clean and straightforward. I like the expert systems paragraph, this is important to understand: ML can be quiet good at specific task (play chess, recognize anomalies in manufacturing processes).

File: content/posts/project/enabling-a-new-class-of-learning-apps.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+date: 2021-12-30
```
Comment (lukekim) [inline content/posts/project/enabling-a-new-class-of-learning-apps.md]:
```suggestion
date: 2022-1-4
```

File: content/posts/project/enabling-a-new-class-of-learning-apps.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+title: "Enabling a new class of learning apps"
```
Comment (lukekim) [inline content/posts/project/enabling-a-new-class-of-learning-apps.md]:
```suggestion
title: "A new class of apps that learn and adapt"
```

File: content/posts/project/enabling-a-new-class-of-learning-apps.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+type: blog
+linkTitle: "Enabling a new class of learning apps"
```
Comment (lukekim) [inline content/posts/project/enabling-a-new-class-of-learning-apps.md]:
```suggestion
linkTitle: "A new class of apps that learn and adapt"
```

**Review by lukekim — APPROVED** [review]
Let's go!

**Pull request merged**